tamr_toolbox/workflow/jobs.py

```diff
@@ -60,6 +63,21 @@ def run(
             LOGGER.error(error_msg)
             raise NotImplementedError(error_msg)
 
+        # Excluding GOLDEN_RECORDS as profile information is refreshed as part of job:
+        # breakpoint()
+        if (
+            (target_type == ProjectType.SCHEMA_MAPPING_RECOMMENDATIONS)
+            | (target_type == ProjectType.CATEGORIZATION)
+            | (target_type == ProjectType.DEDUP)
+        ):
+            if run_profile_unified_datasets:
+                try:
+                    profile = project.unified_dataset().profile()
+                except HTTPError:
```

Comment:
this works really well but I get the following message in the logs even when everything works: 
`Encountered HTTP error code 409. Response body: {"status":409,"class":"com.tamr.common.except.ConflictException","message":"Dataset \"Parts Classification_unified_dataset\" has not been profiled","stackTrace":["com.tamr.common.except.ConflictException::withFormattedMessage::ConflictException.java::39","com.tamr.unify.pub.api.services.DatasetService::fetchDatasetProfile::DatasetService.java::929","com.tamr.unify.pub.api.v1.resources.DatasetSubresource::lambda$getDatasetProfile$10::DatasetSubresource.java::405","com.tamr.unify.pub.api.ExceptionUtils::unwrapSCE::ExceptionUtils.java::36","com.tamr.unify.pub.api.v1.resources.DatasetSubresource::getDatasetProfile::DatasetSubresource.java::405","sun.reflect.NativeMethodAccessorImpl::invoke0::NativeMethodAccessorImpl.java::-2","sun.reflect.NativeMethodAccessorImpl::invoke::NativeMethodAccessorImpl.java::62","sun.reflect.DelegatingMethodAccessorImpl::invoke::DelegatingMethodAccessorImpl.java::43","java.lang.reflect.Method::invoke::Method.java::498","org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory$1::invoke::ResourceMethodInvocationHandlerFactory.java::81","org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1::run::AbstractJavaResourceMethodDispatcher.java::144","org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher::invoke::AbstractJavaResourceMethodDispatcher.java::161","org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker::doDispatch::JavaResourceMethodDispatcherProvider.java::205","org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher::dispatch::AbstractJavaResourceMethodDispatcher.java::99","org.glassfish.jersey.server.model.ResourceMethodInvoker::invoke::ResourceMethodInvoker.java::389","org.glassfish.jersey.server.model.ResourceMethodInvoker::apply::ResourceMethodInvoker.java::347","org.glassfish.jersey.server.model.ResourceMethodInvoker::apply::ResourceMethodInvoker.java::102","org.glassfish.jersey.server.ServerRuntime$2::run::ServerRuntime.java::326","org.glassfish.jersey.internal.Errors$1::call::Errors.java::271","org.glassfish.jersey.internal.Errors$1::call::Errors.java::267","org.glassfish.jersey.internal.Errors::process::Errors.java::315","org.glassfish.jersey.internal.Errors::process::Errors.java::297","org.glassfish.jersey.internal.Errors::process::Errors.java::267","org.glassfish.jersey.process.internal.RequestScope::runInScope::RequestScope.java::317","org.glassfish.jersey.server.ServerRuntime::process::ServerRuntime.java::305","org.glassfish.jersey.server.ApplicationHandler::handle::ApplicationHandler.java::1154","org.glassfish.jersey.servlet.WebComponent::serviceImpl::WebComponent.java::473","org.glassfish.jersey.servlet.WebComponent::service::WebComponent.java::427","org.glassfish.jersey.servlet.ServletContainer::service::ServletContainer.java::388","org.glassfish.jersey.servlet.ServletContainer::service::ServletContainer.java::341","org.glassfish.jersey.servlet.ServletContainer::service::ServletContainer.java::228","io.dropwizard.jetty.NonblockingServletHolder::handle::NonblockingServletHolder.java::49","org.eclipse.jetty.servlet.ServletHandler$CachedChain::doFilter::ServletHandler.java::1623","io.dropwizard.servlets.ThreadNameFilter::doFilter::ThreadNameFilter.java::35","org.eclipse.jetty.servlet.ServletHandler$CachedChain::doFilter::ServletHandler.java::1610","io.dropwizard.jersey.filter.AllowedMethodsFilter::handle::AllowedMethodsFilter.java::45","io.dropwizard.jersey.filter.AllowedMethodsFilter::doFilter::AllowedMethodsFilter.java::39","org.eclipse.jetty.servlet.ServletHandler$CachedChain::doFilter::ServletHandler.java::1610","com.palantir.websecurity.filters.JerseyAwareWebSecurityFilter::doFilter::JerseyAwareWebSecurityFilter.java::63","org.eclipse.jetty.servlet.ServletHandler$CachedChain::doFilter::ServletHandler.java::1610","com.serviceenabled.dropwizardrequesttracker.RequestTrackerServletFilter::doFilter::RequestTrackerServletFilter.java::49","org.eclipse.jetty.servlet.ServletHandler$CachedChain::doFilter::ServletHandler.java::1610","com.tamr.dw.tracing.RequestTracingServletFilter::doFilter::RequestTracingServletFilter.java::73","org.eclipse.jetty.servlet.ServletHandler$CachedChain::doFilter::ServletHandler.java::1610","com.tamr.zookeeper.dw.servicestate.ServiceStateFilter::doFilter::ServiceStateFilter.java::78","org.eclipse.jetty.servlet.ServletHandler$CachedChain::doFilter::ServletHandler.java::1610","org.eclipse.jetty.servlet.ServletHandler::doHandle::ServletHandler.java::540","org.eclipse.jetty.server.handler.ScopedHandler::nextHandle::ScopedHandler.java::255","org.eclipse.jetty.server.handler.ContextHandler::doHandle::ContextHandler.java::1345","org.eclipse.jetty.server.handler.ScopedHandler::nextScope::ScopedHandler.java::203","org.eclipse.jetty.servlet.ServletHandler::doScope::ServletHandler.java::480","org.eclipse.jetty.server.handler.ScopedHandler::nextScope::ScopedHandler.java::201","org.eclipse.jetty.server.handler.ContextHandler::doScope::ContextHandler.java::1247","org.eclipse.jetty.server.handler.ScopedHandler::handle::ScopedHandler.java::144","org.eclipse.jetty.server.handler.HandlerWrapper::handle::HandlerWrapper.java::132","com.codahale.metrics.jetty9.InstrumentedHandler::handle::InstrumentedHandler.java::239","io.dropwizard.jetty.RoutingHandler::handle::RoutingHandler.java::52","org.eclipse.jetty.server.handler.gzip.GzipHandler::handle::GzipHandler.java::753","io.dropwizard.jetty.BiDiGzipHandler::handle::BiDiGzipHandler.java::67","org.eclipse.jetty.server.handler.RequestLogHandler::handle::RequestLogHandler.java::56","org.eclipse.jetty.server.handler.StatisticsHandler::handle::StatisticsHandler.java::174","org.eclipse.jetty.server.handler.HandlerWrapper::handle::HandlerWrapper.java::132","org.eclipse.jetty.server.Server::handle::Server.java::505","org.eclipse.jetty.server.HttpChannel::handle::HttpChannel.java::370","org.eclipse.jetty.server.HttpConnection::onFillable::HttpConnection.java::267","org.eclipse.jetty.io.AbstractConnection$ReadCallback::succeeded::AbstractConnection.java::311","org.eclipse.jetty.io.FillInterest::fillable::FillInterest.java::105","org.eclipse.jetty.io.ChannelEndPoint$1::run::ChannelEndPoint.java::104","org.eclipse.jetty.util.thread.strategy.EatWhatYouKill::runTask::EatWhatYouKill.java::336","org.eclipse.jetty.util.thread.strategy.EatWhatYouKill::doProduce::EatWhatYouKill.java::313","org.eclipse.jetty.util.thread.strategy.EatWhatYouKill::tryProduce::EatWhatYouKill.java::171","org.eclipse.jetty.util.thread.strategy.EatWhatYouKill::run::EatWhatYouKill.java::129","org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread::run::ReservedThreadExecutor.java::383","org.eclipse.jetty.util.thread.QueuedThreadPool::runJob::QueuedThreadPool.java::882","org.eclipse.jetty.util.thread.QueuedThreadPool$Runner::run::QueuedThreadPool.java::1036","java.lang.Thread::run::Thread.java::748"],"causedBy":null,"service":"pubapi"}
`

Is there any way to prevent this error message from showing int he log?